torch/testing/_internal/common_utils.py

```diff
@@ -2803,21 +2803,21 @@ def _run_custom(self, result=None):
                 key = f"{self.__class__.__name__}.{self._testMethodName}"
                 from .dynamo_test_failures import dynamo_expected_failures, dynamo_skips
 
-                def expect_failure(f):
+                def expect_failure(f, test_name):
                     @wraps(f)
                     def wrapper(*args, **kwargs):
                         try:
                             f(*args, **kwargs)
                         except BaseException as e:
                             self.skipTest(e)
```

Comment:
I'd love to also add it here, but not sure if that affects the parsing for passrate and update_failures.